test/gtest/ucp/test_ucp_stream.cc

```diff
@@ -64,3 +64,75 @@ UCS_TEST_P(test_ucp_stream, send_recv_data) {
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_stream)
+
+class test_ucp_stream_many2one : public ucp_test {
+public:
+    test_ucp_stream_many2one() : m_receiver_idx(3), m_nsenders(3) {}
+
+    static ucp_params_t get_ctx_params() {
+        return test_ucp_stream::get_ctx_params();
+    }
+
+    virtual void init() {
+        /* Skip entities creation */
+        test_base::init();
+
+        const ucp_test_param &test_param = GetParam();
+
+        if ("\\self" == test_param.transports.front()) {
```

Comment:
can use skip_loopback() (see example in test_upc_wireup.cc)

Reply:
Forgot, this test creates different entities connection map - many2one. sender == receiver check does not work here.

Reply:
how about adding function ucp_test::is_self(), and use it from ucp_test::init() as well?

Reply:
np